.gitignore

```diff
@@ -4,7 +4,7 @@
 /lib/
 /.nyc_output
 /coverage
-/config.json
+/.config*.json
```

Comment:
Why the rename to start with ".". Our other servers convention do not pre-fix the config file with a "."

Reply:
Just cause it seemed more intentional: I wanted prevent .gitignore from matching `config*.json` to avoid unintentionally ignoring a non-config; having `.config` seems pretty explicit that the file is a config file that shouldn't be tracked. I'm happy change it to `/config*.json` in favor of following set conventions.

Reply:
I am in favor of consistency with our other repositories so let's just make it config.json.

Reply:
After that, the PR is approved and you can merge